README.md

```diff
@@ -32,6 +32,10 @@ Hosts: Artem Zinnatullin [@artem_zin](https://twitter.com/artem_zin), Hannes Dor
 
 ### [Episodes](https://github.com/artem-zinnatullin/TheContext-Podcast/releases)
 
+##### Episode 23: Rise of the Machines
```

Comment:
Shouldn’t we mark these episodes though? I. e. _Episode 23: Rise of the Machines (droidcon Berlin 2018)_.

Reply:
Sure can do it, but what about the previous already published panel discussions, should we also rename them too?

Reply:
I think so. It is a pretty good way to find out when podcast clients update the content (and if they update it at all).